app/models/Campaign.coffee

```diff
@@ -42,7 +42,7 @@ module.exports = class Campaign extends CocoModel
 
   getLevelNameMap: () ->
     levelNameMap = {}
-    @getLevels().models.map((l) => levelNameMap[l.get('original')] = l.get('name'))
+    @getLevels().models.map((l) => levelNameMap[l.get('original')] = utils.i18n(l.attributes, 'name').replace('Course: ', ''))
```

Comment:
Would you be able to provide more context for this change.
Why are we replacing the `Course:` section of the string? Would it still be present in other languages?

Reply:
I just attached pictures on asana

Reply:
for the replace statement
I just copy that from another place, didn't think too much😂
and I found this function only be called in one place

Reply:
I'll have a better look tomorrow. Is there an alternative option that might not require string replacement?

Reply:
In fact, I didn't know much about i18n level name, maybe we should ask for somebody else for help😳😳

Reply:
I think it will just work without using the replace.
`@getLevels().models.map((l) => levelNameMap[l.get('original')] = utils.i18n(l.attributes, 'name'))`

Reply:
I just checked that we are using this replace statement everywhere we are translating the level name. I think I understand why we need the replace statement - its because some level names are prefixed with `Course: `, so this would remove that.
So, I think it's fine to use that here as well.

Reply:
updated:)